modules/TEP.py

```diff
@@ -5,14 +5,12 @@
 import logging
 import sys
 
+from common.evidence import detect_spark_memory_limit, write_evidence_strings
```

Comment:
Local modules come last when sorting imports, at least according to PEP. Are you using any type of formatter?

Reply:
Yes, that's right. I'll fix it.